src/mdsynthesis/limbs.py

```diff
@@ -216,7 +215,11 @@ def _apply_resnums(self):
                 resnums = None
 
         if resnums:
-            self._treant._universe.residues.set_resnums(np.array(resnums))
+            # Compatibility for MDAnalysis pre 0.16.0
```

Comment:
Eventually we want to get rid of the storage of resnums here. The feature is hidden in MDS at the moment because we definitely won't be supporting it in the future, but it was really useful and important to work I was currently doing. With the new topology system in MDA and usage of [persistent topologies](https://github.com/MDAnalysis/mdanalysis/issues/643) we can completely eliminate this soon.

Thanks for this, @kain88-de. We keep it for now.